### ISSUE TITLE
read.tableerror

### ISSUE THREAD

**Issue by SripadJ** [RimGubaev/extract_promoters#1]
Error in read.table(file = file, header = header, sep = sep, quote = quote,  : 
  no lines available in input
Calls: read.csv -> read.table
Execution halted
 I am getting this message when I run the script

**Comment by pkfsantos** [comment]
Hi. I also had this problem, it was not generating the genes.gff file.
So, I manually created the genes.gff file:

awk -F '\t' '$3~/gene/' file.gff > genes.gff

and I omitted the line  "#awk '/gbkey=Gene/' $var2 > genes.gff" in the extract_promoters.sh file.

It worked. Now I will check if the results are what I am expecting.

**Comment by Rohit-Satyam** [comment]
> Hi. I also had this problem, it was not generating the genes.gff file.
> So, I manually created the genes.gff file:
> 
> awk -F '\t' '$3~/gene/' file.gff > genes.gff
> 
> and I omitted the line "#awk '/gbkey=Gene/' $var2 > genes.gff" in the extract_promoters.sh file.
> 
> It worked. Now I will check if the results are what I am expecting.

Yes besides using `/gbkey=Gene/` use the third column using `'$3 == "gene"'` because you would not want to consider pseudogenes while looking for the promoters because those genes are non-functional. The @pkfsantos solution still gives pseudogenes in the output.

**Comment by viteo** [comment]
you might want to replace in the _extract_promoters.sh_
`awk '/gbkey=Gene/' $var2 > genes.gff`
to
`awk -F '\t' '$3~/gene/' $var2 > genes.gff`
